pulp_2_tests/tests/rpm/api_v2/test_content_applicability.py

```diff
@@ -70,6 +70,10 @@
                         'type': 'array',
                         'items': {'type': 'string'}
                     },
+                    'modulemd': {
+                        'type': 'array',
+                        'items': {'type': 'string'}
+                    },
```

Comment:
@kersommoura wasn't sure if we wanted to update the docstring on line 90 with a link to the Story in RM that changed the schema. 

Left that out, but I am fine with adding it if it helps us understand better why this changed.